tests/kafka_consumer_manager/test_utils.py

```diff
@@ -0,0 +1,137 @@
+import mock
+from kafka.common import ConsumerTimeout
+
+from kafka_utils.kafka_consumer_manager.util import KafkaGroupReader
+from kafka_utils.util.offsets import PartitionOffsets
+
+
+class TestUtils(object):
+
+    def test_process_consumer_offset_message_topic_get(self):
+        kafka_config = mock.Mock()
+        kafka_group_reader = KafkaGroupReader(kafka_config)
+
+        assert kafka_group_reader.kafka_groups == {}
+
+        with mock.patch.object(
+            kafka_group_reader,
+            'parse_consumer_offset_message',
+            return_value=[
+                'test_group',
+                'test_topic',
+                0,
+                45
+            ]
+        ):
+            kafka_group_reader.process_consumer_offset_message('test message')
+            assert kafka_group_reader.kafka_groups['test_group'] == {'test_topic'}
+
+    def test_process_consumer_offset_message_topic_pop_no_offset(self):
+        kafka_config = mock.Mock()
+        kafka_group_reader = KafkaGroupReader(kafka_config)
+
+        kafka_group_reader.kafka_groups['test_group'] = {'test_topic'}
+        assert kafka_group_reader.kafka_groups['test_group'] == {'test_topic'}
+
+        with mock.patch.object(
+                kafka_group_reader,
+                'parse_consumer_offset_message',
+                return_value=[
+                    'test_group',
+                    'test_topic',
+                    0,
+                    None
+                ]
+        ):
+            kafka_group_reader.process_consumer_offset_message('test message')
+            assert kafka_group_reader.kafka_groups == {}
+
+    def test_read_groups(self):
+
+        class Mock_Consumer(object):
```

Comment:
We don't really need this class in this test. We should be able to use mock_consumer.return_value.next.return_value = mock.Mock(partition=0, topic='test_topic')